frontend/src/app/modules/fields/edit/field-types/date-edit-field.component.ts

```diff
@@ -30,6 +30,7 @@ import {Component} from "@angular/core";
 import * as moment from "moment";
 import {TimezoneService} from "core-components/datetime/timezone.service";
 import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.component";
+import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0d5d295fec5e57b4542a818c3f681c2be34b856b/frontend/src/app/modules/fields/edit/field-types/date-edit-field.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>